gbn/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ volumes:
 services:
   traefik:
     container_name: bridgehead-traefik
-    image: traefik:latest
+    image: traefik:2.4
```

Comment:
```suggestion
    image: traefik:latest
```

Reply:
Careful, old traefik versions are out of support, see [here](https://doc.traefik.io/traefik/deprecation/releases/). Use `latest` or a new version number (2.7) -- but then you'd have to remember updating the tag. I'd recommend using `latest`. If we're surprised by an incompatible version 3, we can easily rollback.